query_V1

```diff
@@ -1,5 +1,20 @@
 <?php
+		session_start();
+		if(!isset($_SESSION['login_user'])){
+				header("location:login.php");
+				}
+		else{
+			$User = $_SESSION['login_user'];
+		}
+?>
```

Comment:
Here I have now added a session start to the query page. This means that the user MUST log in before accessing the query pages.